metron-platform/metron-indexing/src/test/java/org/apache/metron/indexing/dao/UpdateIntegrationTest.java

```diff
@@ -46,6 +49,7 @@
   private static String index;
   private static MockHTable table;
   private static IndexDao hbaseDao;
+  protected static AccessConfig accessConfig;
```

Comment:
As a general access pattern, I think exposing fields for use via inheritance opens up the possibility for hiding fields and non-explicit side effects. Can you either expose this via a `getAccessConfig()` method or invoke it in the subclass via `super.accessConfig`?

https://docs.oracle.com/javase/tutorial/java/IandI/hidevariables.html